src/components/common/YourCompany/YourCompany.vue

```diff
@@ -5,6 +5,8 @@
       <label class="define-company-title">Your {{ getResource.entityReference }}</label>
     </div>
 
+    <!-- Travis Semple - This div should be refactored into a component.
+         Faiz is working on this part, I've asked him to refactor into a component. -->
     <!-- Business/Company Name + Name Request Applicant -->
     <div id="company-name-section" class="section-container" :class="{'invalid-section': invalidNameSection}">
```

Comment:
This will get refactored as part of https://github.com/bcgov/entity/issues/12894